modelci/types/bo/model_bo.py

```diff
@@ -56,6 +56,7 @@ def __init__(
             dataset: str,
             metric: Dict[Metric, float],
             task: Task,
+            parent_model_id: str,
```

Comment:
Note that this file should do removed in a few releases. We do not follow a Java design pattern anymore for more readability.